Orange/widgets/data/owpreprocess.py

```diff
@@ -1803,6 +1938,8 @@ def __on_modelchanged(self):
     def set_data(self, data=None):
         """Set the input data set."""
         self.data = data
+        self.controler.set_max_num_attributes(len(data.domain.attributes) if
+                                              self.data else None)
```

Comment:
Bad line breaking. Before `if` would be better, or the whole argument in another line. Please don't rely on PyCharm doing too much of your work. :laughing: